test/fixtures/node_tags.yml

```diff
@@ -9,7 +9,7 @@ awesome:
   uid: 2
```

Comment:
Please remove these changes.

Reply:
Please remove these changes.

Reply:
Don't resolve the suggested changes.

Reply:
I mean press the resolve button. You need to untrack this file from the last commit.